package.json

```diff
@@ -50,7 +57,7 @@
     "toml": "~3.0"
   },
   "workspaces": [
-    "antora-ui-camel"
-  ],
-  "packageManager": "yarn@3.1.0"
```

Comment:
Yarn removed it when I updated dependencies, I'm not sure why.  Perhaps because I followed recent instructions for node >= 16.10 and ran `corepack enable`? https://yarnpkg.com/getting-started/install

I recently updated several of my projects to use yarn pnp and some got the packageManager key and some didn't.